### PR TITLE
feat: add joystick icon

### PR DESCRIPTION
**Pull request by mittalyashu**
| Preview |
| --- |
| CleanShot 2022-05-15 at 13 23 52@2x |

### CODE REVIEW

**Comment by karsa-mistmere** [comment]
I find that left side box a bit distracting, here are a few alternate designs:
image

Question: what would the use case be? Is this something we need in addition to `gamepad` and `gamepad-2`?

**Comment by mittalyashu** [comment]
Thank you for sharing the alternate designs @karsa-mistmere 

I like the 1st one though, 2nd and 3rd doesn't seems to align with lucide icon style.

**Comment by karsa-mistmere** [comment]
I guess it's all a matter of perspective (🙃), but yeah, I completely agree with you: the flat style suits Lucide the best.

File: icons/joystick.svg [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path
+    d="M21 17a2 2 0 0 0-2-2H5a2 2 0 0 0-2 2v2a2 2 0 0 0 2 2h14a2 2 0 0 0 2-2v-2Z"
+    stroke="#000"
+  />
+  <path d="M3 17v-4a2 2 0 0 1 2-2h1a2 2 0 0 1 2 2v2" stroke="#000" />
+  <path d="M12 16V9" stroke="#000" />
+  <circle cx="12" cy="6" r="3" stroke="#000" />
```
Comment (karsa-mistmere) [inline icons/joystick.svg:18]:
```suggestion
  <path d="M21 17a2 2 0 0 0-2-2H5a2 2 0 0 0-2 2v2a2 2 0 0 0 2 2h14a2 2 0 0 0 2-2v-2Z"/>
  <path d="M6 15v-2"/>
  <path d="M12 15V9"/>
  <circle cx="12" cy="6" r="3"/>
```

File: tags.json [hunk 2]
```diff
@@ -1210,6 +1210,9 @@
     "code",
     "automation"
   ],
+  "joystick": [
+    "game"
+  ],
```
Comment (karsa-mistmere) [inline tags.json:1215]:
```suggestion
  "joystick": [
    "game",
    "console",
    "control stick"
  ],
```

**Pull request merged**